CloneResponsive/styles.css

```diff
@@ -0,0 +1,812 @@
+  padding: 0px 16px;
+  gap: 8px;
+
+  background: #ffffff;
+  border: 1px solid #eaecf0;
+  border-radius: 6px;
+}
+#search-input {
+  border: none;
+  line-height: 30px;
+  width: 100%;
+}
+.header__main__avatar__icon {
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+
+  background-image: url(./images/header-Icons/avatarMark.jpg);
+  background-size: contain;
+  position: relative;
+}
+.header__main__avatar__icon__status {
+  position: absolute;
+
+  right: 0px;
+  bottom: 0px;
+}
+.content__container__upgrade {
+  background-color: #282828;
+  margin: 32px 32px 32px 264px;
+  padding: 32px 36px;
+  height: 128px;
+  min-width: 375px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  align-self: stretch;
+  border-radius: 16px;
+}
+.content__container__upgrade__description {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 8px;
+}
+.content__container__upgrade__description h3 {
+  font-weight: 600;
+  font-size: 28px;
+  line-height: 34px;
+
+  letter-spacing: -0.6px;
+
+  color: #ffffff;
+}
+.content__container__upgrade__description p {
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 22px;
+
+  color: #ffffff;
+}
+
+.content__container__upgrade__button {
+  width: 143px;
+  height: 46px;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+
+  background: #ffffff;
+  border-radius: 39px;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 46px;
+  gap: 8px;
+  text-align: center;
+  letter-spacing: -0.4px;
+}
+.content__container__upgrade__button__icon {
+  margin-top: 6px;
+}
+
+.content__container__stats {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  margin: 32px 32px 32px 264px;
+  gap: 32px;
+}
+.content__container__stats__revenue,
+.content__container__stats__users,
+.content__container__stats__newSignups,
+.content__container__stats__retention {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: start;
+  background-color: #fff;
+  width: 262px;
+  height: 162px;
+  box-shadow: 0px 1px 10px rgba(0, 0, 0, 0.06);
+  border-radius: 12px;
+  padding: 20px 20px;
+}
+.content__container__stats__revenue p,
+.content__container__stats__users p,
+.content__container__stats__newSignups p,
+.content__container__stats__retention p {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 26px;
+
+  color: #5f6980;
+}
+.content__container__stats__revenue h2,
+.content__container__stats__users h2,
+.content__container__stats__newSignups h2,
+.content__container__stats__retention h2 {
+  font-weight: 600;
+  font-size: 26px;
+  line-height: 40px;
+
+  color: #282828;
+  margin-top: 0;
+  margin-bottom: auto;
+}
+.content__container__stats__revenue__change,
+.content__container__stats__users__change,
+.content__container__stats__newSignups__change,
+.content__container__stats__retention__change {
```

Comment:
It is just one class, and the same for all cases below when you use one style for several elements